backend/expensia/expense/views.py

```diff
@@ -9,3 +9,16 @@ class ExpenseCategoryView(ModelViewSet):
     serializer_class = CategorySerializer
     permission_classes = (AllowAny,)
     authentication_classes = ()
+
+
+class ExpenseViewSet(ModelViewSet):
+    queryset = Expense.objects.all()
+    serializer_class = ExpenseSerializer
+    permission_classes = (AllowAny,)
+    filter_fields = ("start_date", "end_date")
```

Comment:
These will become custom filtering
you can create a FilterClass for them or do it manually in the `filter_queryset `method